test/src/api/assertions/testTextMatches.js

```diff
@@ -0,0 +1,90 @@
+const assert = require('assert');
+const Globals = require('../../../lib/globals.js');
+
+describe('assert.textMatches', function () {
```

Comment:
can we add a test where element is not found?

Reply:
added the test